plugins/outputs/honeycomb/honeycomb.go

```diff
@@ -0,0 +1,133 @@
+package honeycomb
+
+import (
+	"errors"
+	"fmt"
+	"github.com/honeycombio/libhoney-go"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+type Honeycomb struct {
+	APIKey      string   `toml:"apiKey"`
+	Dataset     string   `toml:"dataset"`
+	APIHost     string   `toml:"apiHost"`
+	SpecialTags []string `toml:"specialTags"`
+}
+
+var sampleConfig = `
```

Comment:
```suggestion
const sampleConfig = `
```

We now follow a different scheme to handle `sampleConfig`, which adds a file `sample.conf` containing the config and the embedding this config into the code and the README. You can checkout the [example plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/example) for details.